specification.md

```diff
@@ -0,0 +1,152 @@
+
+```yaml
+project-lifecycle:
+  stage: active
+  roadmap: https://foo.bar/roadmap.html
+  bug-fixes-only: false
+  core-maintainers:
+    - https://github.com/scovetta
+    - joe.bob@email.com
+```
+
+Note that the last value on this list is a conditional requirement, making it optional if the project is no longer being maintained. We’ll see more values like this in the following sections.
+- `stage` (Required): The project’s maintenance stage.
+  - **String**. Values may be active or inactive.
+- `roadmap`: URI to information about the project’s roadmap.
+  - **String** beginning with **https://**
+- `bug-fixes-only`: Is the project active, but only accepting bugfix changes?
+  - **Boolean**
+- `core-maintainers`: Required if bug-fixes-only is true or stage is active. Contact information for the project’s maintainers, such as email or URI.
+  - **List** of **String** entries
+
+## Contribution Policy
+The contribution-policy section describes the decisions made by the project regarding when and how to accept contributions.
+
+```yaml
+contribution-policy:
+  accepts-pull-requests: true
+  accepts-automated-pull-requests: true
+  automated-tools-list:
+    - automated-tool: example/security-research
+      action: denied
+      path:
+        - main/foo/bar
+        - main/examples
+      comment: |
+        Lorem ipsum dolor sit amet, consectetur adipisci elit,
+        sed do eiusmod tempor incidunt ut labore et dolore magna aliqua.
+  contributing-policy: https://example.com/development-policy.html
+  code-of-conduct: https://example.com/code-of-conduct.html
+```
+
+- `accepts-pull-requests` (Required): Does your project accept PRs from non-maintainers?
+  - **Boolean**
+- `accepts-automated-pull-requests` (Required): Does your project accept PRs from bots or automated tools?
+  - **Boolean**
+- `automated-tools-list`: If you have exceptions to accepts-automated-pull-requests, such as allowing select tools or blocking select tools, review the specification for more information about how to list those exceptions here.
+  - **List** of **Objects**
+- `contributing-policy`: URI to your project’s detailed contribution policy.
+  - **String** beginning with **https://**
+- `code-of-conduct`: URI to your project’s code of conduct.
+  - **String** beginning with **https://**
+
+## Documentation
+The documentation section is simply a list of URIs to any security documentation you have provided for stakeholders.
+
+```yaml
+documentation:
+  - http://foo.bar/wiki
+```
+
+This section is not required. Each entry to the list must be a **String** beginning with **https://**.
+
+## Distribution Points
+Similar to the previous section in our YAML, the distribution-points section is simply a list of URIs. This list should include any locations that users can expect to safely retrieve your released software.
```

Comment:
Suggestion add:
URIs can be URLs or [Package URL](https://github.com/package-url/purl-spec)s, either to repositories, package distribution points, or anywhere else.